sabre/payloadBuilder.go

```diff
@@ -0,0 +1,420 @@
+package sabre
```

Comment:
Please use underscores in filenames -- payload_builder.go vs. payloadBuilder.go. This avoids some minor issues with git and filesystems which are not really case sensitive, and keeps consistency across the project and with the Go repos in Sawtooth.